README.md

```diff
@@ -1,14 +1,39 @@
-This repository provides a development environment for Bref via a Docker image.
+This repository provides a development environment for Bref via single Docker image and convenience script.
 
 Thanks to that image you can easily get started with Bref without having to install its tools: `serverless`, `aws`, `php`, `composer`…
 
 ## Usage
 
-Run any command in the container:
+Run any command in the container using `./dev-env.sh`:
 
 ```bash
-docker run --rm -it bref/dev-env <command>
+./dev-env <command>
 
 # For example:
-docker run --rm -it bref/dev-env php -v
+./dev-env php -v
 ```
+
+or via `docker run` directly:
```

Comment:
I think we can skip documenting this. Either users know how to look in the bash file, either they shouldn't mess with all that anyway (because they will open up issues about this).